src/ACF/Field_Section.php

```diff
@@ -50,14 +49,18 @@ public function get_fields(): Field_Collection {
 	}
 
 	/**
-	 * @return Field
+	 * @return array
 	 */
-	public function get_section_field() {
-		//Since a section is really a faux field, we don't really need a real name for it.
-		return new Field( uniqid( 'section-' ), [
-			'type'  => $this->type,
-			'label' => $this->label,
-		] );
+	public function get_attributes() {
+		$field_attributes   = [];
+		$field_attributes[] = $this->attributes;
+
+		/** @var Field $field */
+		foreach ( $this->fields as $field ) {
+			$field_attributes = array_merge($field_attributes, $field->get_attributes());
```

Comment:
`array_merge` inside of a loop is an inefficient construct (see https://github.com/kalessil/phpinspectionsea/blob/master/docs/performance.md#slow-array-function-used-in-loop). The merge can happen once using:

```
public function get_attributes() {
	return array_merge( [ $this->attributes ], ... array_map( static function ( ACF_Configuration $field ) {
		return $field->get_attributes();
	}, iterator_to_array( $this->fields ) ) );
}
``` 

or this slightly more procedural version:

```
public function get_attributes() {
	$field_attributes = [
		[ $this->attributes ],
	];
	/** @var ACF_Configuration $field */
	foreach ( $this->fields as $field ) {
		$field_attributes[] = $field->get_attributes();
	}

	return array_merge( ...$field_attributes );
}
```